docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1538,6 +1539,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/cosFileMaxResources`|500000|
 |`fhirServer/bulkdata/cosFileMaxSize`|209715200|
 |`fhirServer/bulkdata/patientExportPageSize`|200|
+|`fhirServer/bulkdata/useFhirServerTrustStore`|false|
```

Comment:
I think I'd rather see this default to true.  I think its much simpler to have a single trustStore used by the whole server.  Then we should make sure our sample configs don't have trustStore settings sprinkled all over.

Reply:
we can not, set to true will fail S3/IBM COS. because there is no public CAs in our truststore.